recipes/weblogo/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "weblogo" %}
+{% set version = "3.7.8" %}
+{% set license = "MIT" %}
+{% set license_file = "LICENSE.txt" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/weblogo-{{ version }}.tar.gz
+  sha256: c6f8ff118ec0c8401a7d4c3c111c0108eee8523b092afe0dccec42e8134622f3
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - weblogo = weblogo._cli:main
+    - transformseq = weblogo._transformseq:main
+  script_env:
+    ## NOTE:
+    # https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#inherited-environment-variables
+    - SEP_CHAR=/  # [not win]
+    - SEP_CHAR=\\  # [win]
+  script:
+    {% set licenses = [
+      "LICENSE_biopython.txt",
+      "LICENSE_BSD-3-clause.txt",
+      "LICENSE_CC-BY-ND-1.0.txt",
+      "LICENSE_weblogo_color.py.txt",
+      "LICENSE_weblogo_colorscheme.py.txt",
+      "LICENSE_weblogo_data.py.txt",
+      "LICENSE_weblogo_logomath.py.txt",
+      "LICENSE_weblogo_matrix.py.txt",
+      "LICENSE_weblogo_MIT_1.txt",
+      "LICENSE_weblogo_MIT_2.txt",
+      "LICENSE_weblogo_MIT_3.txt",
+      "LICENSE_weblogo_MIT_4.txt",
+      "LICENSE_weblogo_seq.py.txt",
+      "LICENSE_weblogo_seqio.txt",
+      "LICENSE_weblogo_transform.py.txt",
+    ] %}
+    - cp -r ${RECIPE_DIR}/licenses licenses  # [not win]
+    - xcopy /s /e /q /i %RECIPE_DIR%\licenses licenses  # [win]
```

Comment:
You don't need this. `conda-build` will also look for paths in RECIPE_DIR.

Reply:
Tested already. Thank you. Will remove this.